src/app/dock_widgets/process/gt_processdock.cpp

```diff
@@ -1943,7 +1923,7 @@ GtProcessDock::skipComponent(const QModelIndex& index, bool skip)
     QString msg = tr("%1 selected Process Elements")
                   .arg(skip ? tr("Skip") : tr("Unskip"));
 
-    auto cmd = gtApp->makeCommand(gtApp->currentProject(), msg);
+    auto cmd = gtApp->makeCommand(m_taskGroup, msg);
     Q_UNUSED(cmd)
 
     skipComponent(componentByModelIndex(index), skip);
```

Comment:
I would suggest to narrow it down to the object that is skipped
```suggestion
    auto* comp = componentByModelIndex(index);
    if (!comp) return;
    
    auto cmd = gtApp->makeCommand(comp, msg);
    Q_UNUSED(cmd);
    
    skipComponent(comp, skip);
```